csrc/velox/functions/CMakeLists.txt

```diff
@@ -19,4 +21,5 @@ target_link_libraries(
   torcharrow_udfs
   velox_functions_string
   velox_functions_prestosql
+  "${TORCH_LIBRARIES}"
```

Comment:
`TORCH_LIBRARIES` should be added in the funciton's CMakeList. Otherwise `#include <torch/script.h>` will complain.